extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -221,53 +90,52 @@ In order for the video integration to work, the embedded document (e.g. `my-vide
-playback methods and event dispatchers to plug these together. For common video frameworks, the integration script
-[provides readymade playback support](#readymade-integrations), but you can also [write a custom integration yourself](#custom-integrations) if you don't use any of the tools for which support is available.
-
-{% call callout('Do not autoplay yourself', type='caution') %}
-**Never play the video inside the frame automatically.** Instead, you should support the integration script and use the `amp-video-iframe` tag with the `autoplay` attribute. The AMP component will automatically send the necessary signals to your iframe to autoplay for a better user experience.
-{% endcall %}
+Provide a callback that specifies how the AMP document and the iframes video document communicate. You must implement a set of playback methods and event dispatchers to sync the documents together. Implement support by:
 
-### Readymade integrations
+- Using an existing [third party provided readymade integration](#third-party-readymade-integrations).
+- Writing a [custom integration](#custom-integrations) if your solution doesn't use supported tools.
+- [Contributing a third party integration](#contributing-a-third-party-integration) for future use.  
 
-If you're using a common video framework like [JwPlayer](https://www.jwplayer.com/) or [Video.js](http://videojs.com/), you can call **`listenTo()`** for a basic, readymade integration. These integrations support all playback and UI controls when the framework provides them, see each for supported methods.
+#### Third party readymade integrations
 
-{% call callout('Framework APIs', type='note') %}
-Depending on which video framework you use, you'll call the `listenTo` method differently. Read on the specific APIs below.
-{% endcall %}
+Third party video frameworks contributed their integrations for use with `<amp-video-iframe>`.
 
-{% call callout('Expanded support', type='note') %}
-You can additionally use [custom integration methods](#custom-integrations) if you require a feature not available in readymade implementations.
-{% endcall %}
+##### [JwPlayer](https://developer.jwplayer.com/jwplayer/docs)
 
-##### For JwPlayer
-
-**Default supported events:** `ad_end`/`ad_start`, `canplay`, `error`, `muted`/`unmuted`, `pause`/`playing`
-
-**Default supported methods:** `pause`/`play`, `mute`/`unmute`, `hidecontrols`/`showcontrols`, `fullscreenenter`/`fullscreenexit`
-
-Pass in your [`jwplayer` instance object](https://developer.jwplayer.com/jw-player/docs/javascript-api-reference/)
-through the signature `ampIntegration.listenTo('jwplayer', myJwplayer)`. The `ampIntegration` object then knows how
-to setup the player through the instance API.
+Pass your [`jwplayer` instance object](https://developer.jwplayer.com/jw-player/docs/javascript-api-reference/)
+through the signature `amp.listenTo('jwplayer', myJwplayer)`. The `amp` object sets up the player through the instance API.
```

Comment:
After #26969, the second argument here is optional:

```suggestion
The `amp` object sets up a [JwPlayer instance](https://developer.jwplayer.com/jw-player/docs/javascript-api-reference/) by using `listenTo('jwplayer')`. If you're embedding your player using a video-specific script, you only need to register for Jwplayer usage:

```html
<script src="https://cdn.jwplayer.com/players/UVQWMA4o-kGWxh33Q.js"></script>
<script>
  (window.AmpVideoIframe = window.AmpVideoIframe || []).push(amp => {
    amp.listenTo('jwplayer');
  });
</script>
Otherwise, pass in your JwPlayer instance through the signature amp.listenTo('jwplayer', instance):

```js
(window.AmpVideoIframe = window.AmpVideoIframe || []).push(amp => {
  amp.listenTo('jwplayer', jwplayer('my-video'));
});
```

```